### DISCUSSION TITLE
Support for "private" Urls

### DISCUSSION THREAD

**Discussion by mrdavidlaing** [YU000jp/logseq-plugin-confirmation-hyperlink#22] [Q&A]
I add links for many private / internal Urls to my Logseq notes (Logseq Desktop on Mac) - eg, to a Google doc that only myself and people within my company.

<img width="800" alt="image" src="https://github.com/YU000jp/logseq-plugin-confirmation-hyperlink/assets/227505/93b7819e-dc75-44f7-b6ef-77406c8f1ea1">

When I add this GDoc url to my Logseq notes the `logseq-plugin-confirmation-hyperlink` fetches it's title as `Error 403 Forbidden!!1` rather than the document's title.

<img width="610" alt="image" src="https://github.com/YU000jp/logseq-plugin-confirmation-hyperlink/assets/227505/581e7020-b078-41ef-80ad-b20cc87941ae">

It would be great if I could somehow share my browser's (Chrome) session credentials with `logseq-plugin-confirmation-hyperlink` so that t could extract the correct title.

Is this possible?

**Comment by YU000jp** [comment]
Access to private documents that require login, etc., is not possible through the Logseq plugin. This is necessary for security and privacy reasons. If the document is publicly available, the title will be fetched.
Regarding error codes, it is necessary to remove them from the input field, so I will make that correction.

**Comment by YU000jp** [comment]
Perhaps, if the title is a hyperlink, copying and pasting that title into Logseq might create a hyperlink. I have not tried this before.

**Comment by mrdavidlaing** [comment]
Interestingly, if I copy the url from Microsoft Edge and then paste that into Logseq, it automatically creates a markdown link with the (correct) doc title.

![image](https://github.com/YU000jp/logseq-plugin-confirmation-hyperlink/assets/227505/ed0fc518-5256-4d4c-9727-0ed0bbf1becd)

Copying the url from Chrome doesn't do this - rather I get the `logseq-plugin-confirmation-hyperlink` popup 
![image](https://github.com/YU000jp/logseq-plugin-confirmation-hyperlink/assets/227505/96627d25-c1b3-4bcf-bb6b-3f174cbe5987)
Selecting the Doc title sadly doesn't work either
![image](https://github.com/YU000jp/logseq-plugin-confirmation-hyperlink/assets/227505/6f25dab9-c4e5-4bba-a994-f6c0d33c0bd5)

**Comment by mrdavidlaing** [comment]
[This discussion ](https://superuser.com/questions/1286317/how-to-copy-a-url-with-its-title-in-markdown-format)suggests that Chrome can be modified to support this behaviour
